src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/JavaScript.java

```diff
@@ -165,8 +165,7 @@ public Object executeAsyncJavascript(String... code) {
 	public String getAlertMessage() {
 		try {
 			Alert alert = browserManagement.getCurrentWebDriver().switchTo().alert();
-			String text = alert.getText().replace("\n", "");
-			return text;
+			return alert.getText().replace("\n", "");
```

Comment:
Can we use `.replace('\n', '')` instead?

Reply:
it is not working. only alert.getText().replace("\n", "");

Reply:
Yes, I see it now. Thanks for correcting me!